examples/kind/tests/sample_test.go

```diff
@@ -22,8 +22,51 @@ import (
 	"time"
 
 	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/suite"
 )
 
+type Suite struct {
+	suite.Suite
+}
+
+func (s *Suite) TestPass() {
+	logrus.Infof("Passed test:" + os.Getenv("KUBECONFIG"))
+}
+
+func (s *Suite) TestFail() {
+	logrus.Infof("Failed test: " + os.Getenv("KUBECONFIG"))
+
+	s.T().FailNow()
+}
+
+type FailedSuite struct {
+	suite.Suite
+}
+
+func (s *FailedSuite) SetupSuite() {
+	logrus.Infof("Failed suite: " + os.Getenv("KUBECONFIG"))
+
+	s.T().FailNow()
+}
+
+func (s *FailedSuite) TestPass() {
+	logrus.Infof("Passed test:" + os.Getenv("KUBECONFIG"))
+}
+
+func (s *FailedSuite) TestFail() {
+	logrus.Infof("Failed test: " + os.Getenv("KUBECONFIG"))
+
+	s.T().FailNow()
+}
+
+func TestSuite(t *testing.T) {
+	suite.Run(t, new(Suite))
+}
+
+func TestFailedSuite(t *testing.T) {
+	suite.Run(t, new(FailedSuite))
+}
+
```

Comment:
Can we move this to other samples with suites?

Reply:
Yep, added different suite samples to tests.